script/GameObjects/Components/Object/Factory.lua

```diff
@@ -81,13 +84,54 @@ function Factory:updateProduction (prod, dt)
         -- TODO : How to handle failure when a factory finishes a production
         --        for which the output inventory has insufficient capacity?
         assert(self.parent:addItem(output.item, output.count))
+if output.item ~= Item.Energy then
+printf("FACTORY %s produced %d units of %s", self.parent:getName(), output.count, output.item:getName())
+end
+      end
+
+      -- TODO: Make factory trade order timers work
+      --       Until then, after a production run respawn bids and asks per factory inputs and outputs
+      for _, input in prod.type:iterInputs() do
+        for i = 1, input.count * Config.econ.inputBacklog do
+          if input.item == Item.Energy then
+            -- Make sure Energy-requiring factories bid well (to spur production)
+            self.parent.trader:addBid(input.item, 10 + rng:getInt(25, 100))
+          elseif input.item == Item.Waste then
+            -- Make sure Waste-requiring factories bid fairly well (to increase Energy supply)
+            self.parent.trader:addBid(input.item, 5 + rng:getInt(5, 45))
+          else
+            self.parent.trader:addBid(input.item, input.item.energy * 10 + rng:getInt(1, 10)) -- TODO: do smarter bids
+          end
+        end
+      end
+      for _, output in prod.type:iterOutputs() do
+        for i = 1, output.count do
+          if output.item == Item.Energy or output.item == Item.WasteRad or output.item == Item.Waste then
+            -- Make Energy and Waste / Radioactive Waste cheap to buy
+            self.parent.trader:addAsk(output.item, output.item.energy)
+          else
+            -- Asking price should be at least the cost of all the inputs plus a little profit
+            local askprice = 1
```

Comment:
askPrice

Reply:
Change made, update in follow-up commit.